modules/hooks/init.js

```diff
@@ -566,6 +566,29 @@ export default function() {
     });
 
 
+    // Register Partial Channelling
+    game.settings.register("wfrp4e", "partialChannelling", {
+      name: "SETTINGS.PartialChannelling",
+      hint: "SETTINGS.PartialChannellingHint",
+      scope: "world",
+      homebrew: true,
+      config: false,
+      default: false,
+      type: Boolean
+    });
```

Comment:
Why is this setting duplicated?

Reply:
this mistake has been sponsored by our one and only duet Copy & Paste. Fixed